tests/storage/ext/test_ext_sr.py

```diff
@@ -7,13 +7,15 @@
 from lib.commands import SSHCommandFailed
 from lib.common import vm_image, wait_for
 from lib.fistpoint import FistPoint
-from lib.vdi import VDI
 from tests.storage import try_to_create_sr_with_missing_device, vdi_is_open
+from tests.storage.storage import install_randstream, operation_on_vdi, wait_for_vdi_coalesce
```

Comment:
Could you add them in the `tests/storage/__init__.py` import to avoid the double storage like it's done for `vdi_is_open` above.

Reply:
done